components/page.jsx

```diff
@@ -12,6 +12,20 @@ var DevRibbon = (process.env.NODE_ENV === 'production' &&
                 : require('./dev-ribbon.jsx');
 
 var Page = React.createClass({
+  statics: {
+    handlerForPage: function(router, url) {
+      return router.match(url).routes[1].handler;
+    },
+    titleForHandler: function(handler) {
+      var title = 'Mozilla Learning';
+
+      if (handler.pageTitle) {
+        title = handler.pageTitle + ' - ' + title;
```

Comment:
Yeah, I think it's the preferred way b/c that way if the text gets cut off, e.g. how long titles get cut off on browser tabs, the text that makes the tab "different" from other tabs on the same site is visible.